views/mailingEdit.html

```diff
@@ -75,12 +75,12 @@ <h3>Workflow</h3>
 		<ul>
 			<li >Select an approver: (<a href="0/approval">send to everyone</a>)
 				<ul>
-				{{#mApprovers}}
+				{{#approvers}}
 					<li> <a href="{{subscode}}/approval" value="{{subscode}}">
 						   {{name}}
 					   </a>
 					</li>
-				{{/mApprovers}}
+				{{/approvers}}
 				</ul>
 			</li>

```

Comment:
Can you add the condition where we are not displaying approvers if there none set? Like I suggested in the code sample in APPS-120, it is the "Change request item 3". Thanks